### ISSUE TITLE
字符检测训练问题

### ISSUE THREAD

**Issue by ff12343** [PaddlePaddle/PaddleOCR#7013]
det_r50_vd_sast 预训练模型训练图片，Eval精度越训练越低
[2022/07/27 15:16:12] ppocr INFO: cur metric, precision: 0.7, recall: 0.0030345798816568047, hmean: 0.005830903790087463
5, fps: 45.4276415849069 (Step: 2000)
[2022/07/27 15:36:12] ppocr INFO: cur metric, precision: 0.2, recall: 0.0029585798816568047, hmean: 0.005830903790087463
5, fps: 45.8224158898691 (Step: 9000)
[2022/07/27 16:05:23] ppocr INFO: cur metric, precision: 0.07692307692307693, recall: 0.0029585798816568047, hmean: 0.00
5698005698005697, fps: 45.675872283901306 (Step: 13000)
.
.
.

后面precision归0了，最后训练模型predict结果如下：
![MyImage-20220118101822189](https://user-images.githubusercontent.com/42758082/181199973-2c00b552-b34d-4cef-a35e-b72ce6fc1f4b.jpg)
标注使用的是PPOCRLabel导出的SampleDataset，格式如下
.\MyImage-20220118130327253.jpg [{"transcription": "FHSK", "points": [[21, 6], [123, 6], [123, 51], [21, 51]], "difficult": false}]

贴出训练参数：
Global:
  use_gpu: true
  epoch_num: 500
  log_smooth_window: 20
  print_batch_step: 2
  save_model_dir: ./output/sast_r50_vd_ic15/
  save_epoch_step: 25
  eval_batch_step:
  - 5000
  - 4000
  cal_metric_during_train: false
  pretrained_model: ./model/sast_det_model/det_r50_vd_sast_icdar15_v2.0_train/best_accuracy
  checkpoints: null
  save_inference_dir: null
  use_visualdl: false
  infer_img: null
  save_res_path: ./output/sast_r50_vd_ic15/predicts_sast.txt
Architecture:
  model_type: det
  algorithm: SAST
  Transform: null
  Backbone:
    name: ResNet_SAST
    layers: 50
  Neck:
    name: SASTFPN
    with_cab: true
  Head:
    name: SASTHead
Loss:
  name: SASTLoss
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    learning_rate: 0.001
  regularizer:
    name: L2
    factor: 0
PostProcess:
  name: SASTPostProcess
  score_thresh: 0.5
  sample_pts_num: 2
  nms_thresh: 0.2
  expand_scale: 1.0
  shrink_ratio_of_width: 0.3
Metric:
  name: DetMetric
  main_indicator: hmean
Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data
    label_file_list:
    - ./train_data/train_data_ff/det/train.txt
    - ./train_data/train_data_lc/det/train.txt
    - ./train_data/train_data_lx/det/train.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - SASTProcessTrain:
        image_shape:
        - 512
        - 512
        min_crop_side_ratio: 0.3
        min_crop_size: 24
        min_text_size: 4
        max_text_size: 512
    - KeepKeys:
        keep_keys:
        - image
        - score_map
        - border_map
        - training_mask
        - tvo_map
        - tco_map
  loader:
    shuffle: true
    drop_last: false
    batch_size_per_card: 4
    num_workers: 4
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data
    label_file_list:
    - ./train_data/train_data_ff/det/val.txt
    - ./train_data/train_data_lc/det/val.txt
    - ./train_data/train_data_lx/det/val.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - DetResizeForTest:
        resize_long: 1536
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - shape
        - polys
        - ignore_tags
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 1
    num_workers: 2
profiler_options: null

**Comment by ff12343** [comment]
测试了其他几个算法，发现基于分割的DB算法不受影响，基于检测的sast，east算法都会出现这个现象，精度越训练越低的情况

**Comment by LDOUBLEV** [comment]
看你的数据，文字部分已经占了全图的绝大部分区域，可以不用检测，直接用识别模型去识别就行

**Comment by Tavish77** [comment]
我也出现这种情况 请问怎么解决呢
